addons/l10n_es_edi_sii/data/account_tax_data.xml

```diff
@@ -176,6 +176,22 @@
         <field name="name">0.50% Recargo Equivalencia Ventas</field>
         <field name="l10n_es_type">recargo</field>
     </record>
+    <record id="l10n_es.account_tax_template_s_req0" model="account.tax.template">
+        <field name="name">0% Recargo Equivalencia Ventas</field>
+        <field name="l10n_es_type">recargo</field>
+    </record>
+    <record id="l10n_es.account_tax_template_p_req0" model="account.tax.template">
+        <field name="name">0% Recargo Equivalencia Compras</field>
+        <field name="l10n_es_type">recargo</field>
+    </record>
```

Comment:
it might be better to move this part above `<record id="l10n_es.account_tax_template_s_req05" model="account.tax.template">` so it's ordered